pymtl3/datatypes/helpers.py

```diff
@@ -70,9 +75,25 @@ def get_nbits( Type ):
   return sum(get_nbits(v) for v in Type.__bitstruct_fields__.values())
 
 def to_bits( obj ):
+  if isinstance( obj, list ):
+    return concat( *[ to_bits(x) for x in obj ] )
+
   # BitsN
   if isinstance( obj, Bits ):
     return obj
   # BitStruct
   assert is_bitstruct_inst( obj ), f"{obj} is not a valid PyMTL Bitstruct!"
   return concat( *[ to_bits(getattr(obj, v)) for v in obj.__bitstruct_fields__.keys() ] )
+
+def _get_bitstruct_inst_all_classes( obj ):
```

Comment:
we probably should allow designers use this (in tests) if they want? Maybe remove the underscore?

Reply:
yeah we could